apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -186,17 +186,35 @@ int ble_rmc_main(int argc, char *argv[])
 	RMC_LOG(RMC_TAG, "cmd : %s\n", argv[1]);
 
 	if (strncmp(argv[1], "init", 5) == 0) {
-		ret = ble_manager_init(&client_config, &server_config);
+		ret = ble_manager_init(NULL);
+		if (ret != BLE_MANAGER_SUCCESS) {
+			RMC_LOG(RMC_TAG, "init fail[%d]\n", ret);
+			return 0;
+		}
 		RMC_LOG(RMC_TAG, "init done[%d]\n", ret);
+	}
 
-		sem_init(&g_conn_sem, 0, 0);
+	if (strncmp(argv[1], "init2", 6) == 0) {
```

Comment:
I think `init2` is not easy to understand. How about changing the command?

Reply:
ok, i will add "init null" for this